README.md

```diff
@@ -5,11 +5,12 @@ node.js command line bitcoin HD wallet, BIP 44 compatible.
 
 ## Requirements
 
-### NPM modules
+Until we have posted the module in the NPM registry, for installing it globally using the command `npm install -g waljs`,
```

Comment:
Remove this phrase. We could change the README again once we publish waljs as a package.